plugins/meta/portmap/portmap.go

```diff
@@ -224,6 +224,16 @@ func fillDnatRules(c *chain, config *PortMapConf, containerIP net.IP) {
 	// the ordering is important here; the mark rules must be first.
 	c.rules = make([][]string, 0, 3*len(entries))
 	for _, entry := range entries {
+		// If a HostIp is given, only process the entry if host and container address families match
+		if entry.HostIP != "" {
+			hostIp := net.ParseIP(entry.HostIP)
```

Comment:
Please name this variable as `hostIP`.
PTAL on https://github.com/golang/go/wiki/CodeReviewComments#initialisms